src/extractor/scripting_environment_lua.cpp

```diff
@@ -47,6 +49,11 @@ template <class T> double lonToDouble(T const &object)
     return static_cast<double>(util::toFloating(object.lon));
 }
 
+template <class T> osmium::Location locationForNode(T const &object)
+{
+        return LuaScriptingEnvironment::extraction_containers->getLocationForNode(object);
+}
```

Comment:
The reason this only works with a static var is the locationForNode function needs to be a member function as in

    struct ScriptingEnv {

    Location getLocation(Node n) {
      return cache.at(n);
    }

      Cache cache;
    }